osmosis-1/osmosis-frontier.assetlist.json

```diff
@@ -1305,6 +1305,36 @@
       },
       "coingecko_id": "switcheo"
     },
+    {
+      "description": "The native EVM, governance and staking token of the Evmos Hub",
```

Comment:
Can we please correct the asset description in Chain Registry to this?
https://github.com/cosmos/chain-registry/blob/master/evmos/assetlist.json